src/CommandingOfficers/CommanderVenge.java

```diff
@@ -61,6 +61,20 @@ public void endTurn()
     aggressors.clear();
   }
 
+  @Override
+  public char getSymbol(Unit unit)
```

Comment:
Instead of overriding and re-implementing the function, I'd prefer more of this to be moved into Commander, I think. Have a Map (you love those, right) of Units to the markers, and just have `add/removeUnitMarker` functions subclasses can call. It shouldn't be necessary to override `getSymbol` at all that way.

Reply:
There should also be at least a comment (and maybe an error) informing/enforcing that only valid characters can be used (Currently [A-Z0-9], no lowercase). Should we accept lowercase and just pretend it was uppercase?

Reply:
I guess a reasonable priority might be:
Owner's mark
my mark
anybody else's mark

Then we could just pass in the list of COs and let 'em go to town.
Then again, if we're going so far as to store all that jazz manually, maybe we should provision for showing all applied marks with cycling or something.

Reply:
I wasn't trying to change this to accommodate displaying anybody's mark; I guess you could, but I'm not sure that's what we actually want.

All what jazz? It would be a map of unit to mark, which you have to have already anyway.
Cycling marks is not a bad idea.

Reply:
I don't have to have any such map, I only need a CO's domain-specific knowledge and a method to convert that to real data, if needed.
Having a map of unit:mark means that there are now two places to keep synched with what should be marked; the domain, and the marks.